src/thorin/util/stream.h

```diff
@@ -12,7 +12,11 @@ class Streamable {
 public:
     virtual ~Streamable() {}
 
-    virtual std::ostream& stream(std::ostream&) const = 0;
+#ifdef _MSC_VER
+    virtual std::ostream& stream(std::ostream& os) const { return os; } // = 0;
+#else // _MSC_VER
+    virtual std::ostream& stream(std::ostream& os) const = 0;
+#endif // _MSC_VER
     std::string to_string() const; ///< Uses @p stream and @c std::ostringstream to generate a @c std::string.
```

Comment:
This was probably an issue similar to Primop::vrebuild() about abstract classes used as template argument for hashing. However, it seems to be no issue anymore and commit 07e0152 removes the special handling.